CHANGELOG.md

```diff
@@ -0,0 +1,89 @@
+# Changelog
+
+All notable changes to Lace will be documented in this file.
+
+## [2.0.0] 
+
+This is a major rewrite of the Lace API. The goal is to make the framework more intuitive to work with by reducing the amount of macro-style code. For example, instead of `SPAWN(fib, 40)` we write `fib_SPAWN(lace, 40)`. In most cases, the performance is not changed by this, except the `knapsack` benchmark, as in the new style the head of the deque is now stored in the worker struct in the heap, rather than as a register or on the program stack. See also [#10](https://github.com/trolando/lace/pull/10).
+
+### Added
+
+- Documentation is much improved now.
+- Added a more 'proper' Changelog (this file).
+- Added a large number of benchmarks from the Nowa repository.
+
+### Changed
+
+- Significant API-breaking rewrite to be a bit more user-friendly.
+- Changed algorithm for `lace_rng` from lcg to lehmer64.
+- `lace_start` now takes three parameters: the number of workers, the size of the task deque, the size of the program stack.
```

Comment:
So, one would have to be able to know a priori how much stack space (in bytes) is used in total and/or by each worker? The *zero* default should maybe be documented here too.